README.md

```diff
@@ -1,68 +1,24 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# muenster-info-app
 
-## Available Scripts
+## Project info
 
-In the project directory, you can run:
+Alles über Münster im Überblick. Hervorgegangen aus dem [MÜNSTERHACK 2019](https://www.muensterhack.de) ist das Münster Info Hub *münster.jetzt,* eine Plattform, um Daten unterschiedlichen Ursprungs an einem einzigen Ort zusammenzuführen. Offen für alle Entwickler und alle erdenkbaren Daten - Hauptsache Münster!
```

Comment:
"Alle erdenkaren Daten"? Ui is das nicht etwas dick aufgetragen? :)

Reply:
Ja genau. Mein letzter Stand war dass wir erstmal nur Veranstaltungen machen. @jahnique hat den Text von www.muenster.jetzt